.github/workflows/ci.yml

```diff
@@ -201,7 +201,8 @@ jobs:
       matrix:
         rust-version: [
           1.72.0, # current stable
-          beta
+          beta,
```

Comment:
Can you comment out beta and stable here, delete the alias and just run `cargo clippy` in this job?

Reply:
@thomaseizinger Just now commit the change. Please check